alarms/Groupme/groupme_alarm.py

```diff
@@ -0,0 +1,89 @@
+#Setup Logging
+import logging
+log = logging.getLogger(__name__)
+
+#Local modules
+from ..alarm import Alarm
+from ..utils import *
+#Bot Implementation Requirements
+import requests
+from json import dumps as jsondumps
+
+
+class Groupme_Alarm(Alarm):
+  # notice: groupme does NOT support percent signs (%) in the message.
+  # I do not know how to fix this. Somebody fix this.
+  _defaults = {
+  'pokemon':{
+  'location_name':"<pkmn> until <24h_time>",
+      'text':"<pkmn> here until <24h_time> (<time_left>).\n IV: <atk>/<dfs>/<sta> \n Move 1: <move1> \n Move 2: <move2>"
```

Comment:
We don't support IV's (yet) and when we do it will be optional. This needs to be more in line with the the other defaults at the moment.